free-2.md

```diff
@@ -11,41 +11,62 @@ _What need is met by your idea? Who are you helping? What is that person's
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+- I like pixel art, and compared to the other idea I wanted to come up with something artisitic for this idea. The exiting thing is digging into pixel art holdheartedly (and likely video game and animation filters. 
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+- Pixel animation sprite sheet generation
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+- I think this could *ideally* be part of a bigger pixel art software (and I think some already exist) but a quick and easy drap and drop exterior coding option is where I would want to go with it. I think its the right way to interact because we want to make it part of an existing workflow, so avoiding pulling up a python document for example is much more significant then some (pregenerated) files that the initial sprite can be droped into. 
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+-Errors are infinite loops, but thats mainly it. I think when the program runs it just generates all the images and presents them as soon as they are ready. Then they can be downloaded seperately or in a sprite sheet with the original. The user requestes how many they want. 
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+- Easy: generate a bunch of sprites with small adjustments in a sheet. 
+- medium: make significant edits to each before exporting - cleaning up-- possibly also moving and placing each new sprite on the generated sheet (ie where the user wants) 
+- probably actually making the original pixel art. 
+
 ### Related work
 
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+- I couldnt find any existing DSL's specifically on pixel art but there are a *lot* of pixel art software (specialized beyond general art programs like photoshop) such as GIMP and paint.net. https://blog.felgo.com/game-resources/make-pixel-art-online
+- I think they address the need very well, and with normal copy paste generating sprite sheets is built in. For this DSL to be worthwhile it would have to be **Faster** then control c control v 
```

Comment:
I feel like this would be a lot of work to improve something that is already very fast.